modules/unsupported/geobuf/pom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.geotools</groupId>
+        <artifactId>unsupported</artifactId>
+        <version>14-SNAPSHOT</version>
+    </parent>
+    <groupId>org.geotools</groupId>
+    <artifactId>gt-geobuf</artifactId>
+    <packaging>jar</packaging>
+    <name>Geobuf DataStore</name>
+    <description>A geobuf DataStore</description>
+    <dependencies>
+        <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-api</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-data</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-cql</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.geotools</groupId>
+            <artifactId>gt-epsg-hsql</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.google.protobuf</groupId>
+            <artifactId>protobuf-java</artifactId>
+            <version>2.6.1</version>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>4.11</version>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+    <build>
+        <plugins>
+            <!--
+            Commented out to make building easier (no dependency on protoc executable.
+            To regenerate the org.geotools.data.geobuf.Geobuf Java file, place the new geobuf.proto
+            in src/main/protobuf and add package org.geotools.data.geobuf; to the first line. Then
+            uncomment the following plugin and run mvn clean install.  This will create a new Geobuf
+            Java file in target/generated-sources.
+            -->
```

Comment:
Rather than this manual procedure consider using a profile.

Or you can do what we do for oracle and use -D property to activate one profile when true, and another when false.